plugins/woocommerce/templates/single-product-reviews.php

```diff
@@ -75,8 +75,8 @@
 					'title_reply'         => have_comments() ? esc_html__( 'Add a review', 'woocommerce' ) : sprintf( esc_html__( 'Be the first to review &ldquo;%s&rdquo;', 'woocommerce' ), get_the_title() ),
 					/* translators: %s is product title */
 					'title_reply_to'      => esc_html__( 'Leave a Reply to %s', 'woocommerce' ),
-					'title_reply_before'  => '<span id="reply-title" class="comment-reply-title">',
-					'title_reply_after'   => '</span>',
+					'title_reply_before'  => '<h3 id="reply-title" class="comment-reply-title">',
+					'title_reply_after'   => '</h3>',
```

Comment:
I worry this might break existing store styles, considering they might have some default styling for `<h3>` elements.

An alternative approach to add semantics to this element would be to use ARIA attributes. Something along these lines: `<span role="heading" aria-level="3">`.

That wouldn't be ideal if it was a new template, but I think it's the safest way to add semantics to the element without the risk of breaking existing themes.

What do you think?

Reply:
That is a smart work around if we want to keep backward compatibility. With this change there is a less chance to break something.

Reply:
That sounds good to me, I will get it updated!